### ISSUE TITLE
IDEA: have all items and costumes for custom blooks

### ISSUE THREAD

**Issue by guycoder3** [Minesraft2/Blooket-Cheats#770]
### Gamemode

global

### Function

The script will allow you to use any item or costume when creating a custom blook.

### FaQ

I have read the FaQ and my suggestion has not already been asked for or isn't possible

**Comment by orangemonkey01** [comment]
I don't think this can happen he had it for a while and took it down cause it didn't work

**Comment by Minesraft2** [comment]
Yea i tried this and it stopped working
